src/ApiFunctions/GetIssuesByProject.ts

```diff
@@ -0,0 +1,93 @@
+export interface Issue {
+    project: string,
+    id: string,
+    summary: string,
+    description?: string,
+    assignee?: {
+      displayName: string,
+      accountId: string
+    },
+    reporter?: {
+      displayName: string,
+      accountId: string
+    },
+    priority: "Highest" | "High" | "Medium" | "Low" | "Lowest",
+    created: string,
+    duedate?: string,
+    timespent?: number
+};
+
+//Pulls Issue data of a given project and returns a promise with the data in a useable format
+export default async function getIssuesByProject(projectKey: string) : Promise<Issue[]> {
+    require('dotenv').config();
+    const username: string | undefined = process.env.ATLASSIAN_USERNAME;
+    const password: string | undefined  = process.env.ATLASSIAN_API_KEY;
+    const domain: string | undefined  = process.env.DOMAIN;
+    let doneBool = false;
```

Comment:
field type should not be included in the variable name. In this case, it should be named as isDone or isCompleted not doneBool.